src/buildApplication/fixPrerenderedRoutes.ts

```diff
@@ -98,7 +99,10 @@ async function getRouteDest(
 	const destRoute = normalizePath(
 		join(
 			dirName,
-			fallback.fsPath.replace(/\.prerender-fallback(?:\.rsc)?(?:\.body)?/gi, '')
+			fallback.fsPath.replace(
+				/\.prerender-fallback(\.((rsc)|(body)|(json)))?/gi,
```

Comment:
This regex feels unnecessarily complex to me, would something like this work instead?
```suggestion
				/\.prerender-fallback\.(?:rsc|body|json)?/gi,
```

Reply:
No. For prerendered html files that would strip out the final dot.

Reply:
ah ok sorry you're right 😓 

than I think this should ok:
```suggestion
				/\.prerender-fallback(?:\.(?:rsc|body|json))?/gi,
```
(you can keep or remove the `?:`s I am a bit fine with both versions)

(or do you think that the extra parens in your version make the regex more clear?)

Reply:
I think that the extra brackets make it a little more clear that it is that specific group of letters, but honestly, I don't really mind. I'll go with your suggestion.

Reply:
ah ok thanks a lot, I prefer not to have them as it makes me wonder if they are there for a specific purpose

(but yeah I'd be ok with having them if you strongly preferred them :) )